jumanji/training/setup_train.py

```diff
@@ -79,7 +80,7 @@ def setup_logger(cfg: DictConfig) -> Logger:
 
 def _make_raw_env(cfg: DictConfig) -> Environment:
     env: Environment = jumanji.make(cfg.env.registered_version)
-    if isinstance(env, Connector):
+    if isinstance(env, Tetris):
```

Comment:
This is wrong. Tetris is not a multi-agent environment.
```suggestion
    if isinstance(env, Connector):
```